### ISSUE TITLE
High-res + Widescreen for Software mode?

### ISSUE THREAD

**Issue by OpenRift412** [atsb/strife-ve-ex#1]
I can't say I have a lot to complain about with Veteran Edition, but the one thing that it really needs is a more modern presentation for its  software mode. I mean the 320x200 emulation is cool and all but if I wanted that I could just play in DOS. Would it be feasible to add the ability to run software mode in the game's set resolution and in widescreen, akin to OpenGL mode?

**Comment by atsb** [comment]
Not really.  The entire renderer is OpenGL.  Adding back a software renderer would be identical to using Crispy Strife (without the widescreen stuff or VE data).

Making a widescreen software renderer isn’t very simple and the whole point of the VE stuff is for it to be enhanced in some way.  You’ll be better off petitioning Fabian for a proper Crispy Strife :)

**Comment by OpenRift412** [comment]
> Not really. The entire renderer is OpenGL. Adding back a software renderer would be identical to using Crispy Strife (without the widescreen stuff or VE data).
> 
> Making a widescreen software renderer isn’t very simple and the whole point of the VE stuff is for it to be enhanced in some way. You’ll be better off petitioning Fabian for a proper Crispy Strife :)

I guess what I meant to say is that in VE, if you disable the "High Quality" option in the Graphics Settings, it goes into a 320x200 256-color mode that closely emulates the software renderer. So what I'm suggesting is, would it be possible to have the "High Quality Off" mode run in native resolution in widescreen?

**Comment by atsb** [comment]
Ahhhhh!!!

I’ll take a look

**Comment by atsb** [comment]
So it looks like the original software mode is kept intact 100%, the closest you could get would be to turn off all the fancy effects but keep the GL renderer active.

**Comment by OpenRift412** [comment]
> So it looks like the original software mode is kept intact 100%, the closest you could get would be to turn off all the fancy effects but keep the GL renderer active.

So does that mean nothing can really be done with the software renderer then?

**Comment by atsb** [comment]
Only if someone adds widescreen support to the software renderer..  I'll see how much code I can bring over from Sprinkled Doom.  It is fairly similar to Chocolate Strife..  I'll keep this open so it is a constant reminder :)
